book/website/collaboration/research-infrastructure-roles/data-wrangler.md

```diff
@@ -0,0 +1,85 @@
+The terminology used to describe various data science activities (for example data collection, wrangling, analysis, modelling, visualisation), and how they map onto different job titles, is domain specific and will vary on a project and organisational level.
+
+(cl-infrastructure-data-wranglers-tasks)=
+## What do Data Wranglers do?
+
+In a data science project, it is commonly observed that data wrangling tasks take the majority of time {cite:ps}`anaconda2020sods`, in contrast to data analysis and modelling.
+Traditionally, data wrangling tasks involve cleaning, restructuring and filtering data into standardised research ready formats.
+However, in terms of Data Wrangler as a profession, day to day tasks and objectives can be much more diverse.   
+
+Data Wranglers work primarily in the space between data generators and data analysts, who are addressing the research question of interest.
+Understanding the intended use for the data in the context of the analysis and research questions, Data Wranglers can be in the position of influencing data generators in improving data collection methods, ultimately benefiting all points of the data life cycle.
+Similarly, Data Wranglers will conduct preliminary analysis on the data to ensure both completeness of data and preparation for data analysis, acting as a proxy for the data generator’s knowledge during the data analysis process.
+A key focus of a Data Wrangler’s role is the preparation of research ready data {cite:ps}`stewart2022table` in which data security, data management and FAIR standards {cite:ps}`mons2017fair` are all core priorities.
+
+### Examples of day-to-day tasks:
+
+* Data retrieval, querying, data analysis and visualizations
+* Data pipeline development, maintenance and improvement
+* Communicating with domain and technical experts
+* Data privacy and infrastructure setup and management
+* Summarising domain specific research papers
+* Sharing and reviewing resources and code
+* Maintaining communication across multiple teams
+* Running workshops on data wrangling and related skills
+
+(cl-infrastructure-data-wranglers-skills)=
+## What qualifications or skills do you need to be a Data Wrangler?
+
+Data Wranglers should have experience with programming (no specific language required, but there is a wider adoption of both `R` and `Python`), database querying (`SQL`) and data analysis.
+They will have an educational background to equip them to engage with the specific research data objects relevant to the projects they will work on; they will therefore have undergraduate and postgraduate degrees, or equivalent experience, giving a good grounding in these areas of study.
+As with many data science and research infrastructure roles, further relevant training and specialisation is understood to happen on the job.
+Additionally, a Data Wrangler will need to have great communication, problem solving and interpersonal skills, a curiosity and willingness to learn.
```

Comment:
```suggestion
Additionally, a Data Wrangler will need to have communication, problem solving and interpersonal skills, a curiosity and willingness to learn.
```

Reply:
I'd probably leave out the communication skills out here since you touch upon those in the next sentence.

Reply:
Rephrased as 

> "They need good problem solving skills, with a curiosity and willingness to learn.
> Lastly, good interpersonal skills are required in order to work with people with many different backgrounds, skillsets and priorities."